apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/TransactionDaoImpl.java

```diff
@@ -324,6 +324,28 @@ public int getTransactionCount() {
         }
     }
 
+    @Override
+    public Long getTransactionCount(TransactionalDataSource dataSource, int from, int to) {
+        if (dataSource == null) {
+            // select from main db
+            dataSource = databaseManager.getDataSource();
+        }
+        try (Connection con = dataSource.getConnection()) {
+            PreparedStatement pstmt = con.prepareStatement("SELECT count(*) as transactionCount FROM transaction WHERE height >= ? AND height < ?");
```

Comment:
Maybe simplify to count(*) without `as transactionCount` and then `getLong(1)`?

Reply:
Done